WcaOnRails/app/models/registration.rb

```diff
@@ -180,7 +180,7 @@ def to_wcif
 
   def self.wcif_json_schema
     {
-      "type" => "object",
+      "type" => ["object", "null"], # Note: for now there may be WCIF persons without registration.
```

Comment:
I'm sorry, but I don't actually remember why this is? Is this some trickiness to support delegates and organizers who aren't actually registered to compete? If we have an open issue talking about this, it would be great to link to it here. If not, then :man_shrugging:

Reply:
Yeah, organizers and delegates end up in the `Persons` array no matter they registered or not, so they may have no registration. https://github.com/thewca/worldcubeassociation.org/blob/239941c0b99234a96386a60804b04e66d11784b9/WcaOnRails/app/models/competition.rb#L1099-L1105
There is no issue open for that, but there was some discussion at the end of #1568. I can add a link to this issue if you consider it worth (?) =)

Reply:
Bump =)